src/jit/lsra.cpp

```diff
@@ -4188,12 +4001,29 @@ void LinearScan::buildRefPositionsForNode(GenTree*                  tree,
         // but not push it
         if (!noAdd)
         {
-            locationInfoList.Append(listNodePool.GetNode(defLocation, interval, tree, (unsigned)i));
+            LocationInfoListNode* nodeToAdd;
```

Comment:
Looks like this and a few other changes made `locationInfoList` local variable useless.